src/MSBuild.Sdk.SqlProj.Templates/templates/inlinefunc/.template.config/template.json

```diff
@@ -6,9 +6,29 @@
     "name": "Inline function",
     "shortName": "inlinefunc",
     "tags": {
-      "language": "SQL",
+      "language": "C#",
```

Comment:
I'm curious about the reasoning for setting language to C# instead of SQL. I know we use a `.csproj` file, but I would still personally expect to find these items under SQL. Can you explain further why the language change is needed?

Reply:
Good question! When using "SQL" no items appear in the dialog - I assume it has to do with how the dotnet support for Add New Item is implemented (with a focus on C#) - note that also items like XML files and json files are in this "category"